src/skin/frontend/base/default/css/factfinder/factfinder.css

```diff
@@ -418,3 +418,7 @@ ff-paging {
     color: #636363;
     border-color: #ccc;
 }
+
+#ff-nothing-found {
```

Comment:
I don't think we need this one

Reply:
if this container won't have `display: none` by default it will be visible for a moment, until records from FACT-Finder will render, which was one of the customer complaints.

Reply:
I just noticed you push the fix to the branch with WC upgrade
https://github.com/FACT-Finder-Web-Components/magento1-module/pull/67/commits/742e825bbba04da01f0cf1357b7f60171e0cb1e4

I' removed this